torch/_export/pass_infra/proxy_value.py

```diff
@@ -35,7 +37,7 @@ def is_tensor(self) -> bool:
         return isinstance(self.data, torch.Tensor)
 
     # pyre-ignore
-    def __iter__(self):
+    def __iter__(self) -> Iterator[object]:
```

Comment:
Should still be 'Iterator[_T]', doesn't solve the typing issue in the other file, this probably just unmasking bad typing elsewhere

Reply:
Typing of the `__init__` was incorrect, you are holding an Iterable, which can be cast to an Iterator multiple times.